channels/us.m3u

```diff
@@ -3564,3 +3564,21 @@ http://wttv-lh.akamaihd.net:80/i/WTTVBreaking_1@333494/master.m3u8
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="Mohabat TV" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/tucTC1O.jpg" group-title="",Mohabat TV
 http://media.mohabat.tv:1935/live_transcoder/ngrp:mohabat.stream_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Ind TV USA
+https://t06858-lh.akamaihd.net/i/t06858a_1@719164/index_1500000_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Jasmin channel 18+ HD
+http://109.71.162.112:1935/live/hd.jasminchannel.stream/chunklist_w1793148297.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Jasmin TV
+http://109.71.162.112:1935/live/hd.jasminchannel.stream/chunklist_w359453841.m3u8
```

Comment:
Jasmin TV channel, as far as I know, broadcasts from Luxembourg.

Reply:
+ IP verification proves it : 
![image](https://user-images.githubusercontent.com/30985701/117485171-5a37d480-af68-11eb-8d76-02cd9efaa511.png)